src/sql-editor/components/SQLEditor.tsx

```diff
@@ -96,6 +99,7 @@ export const SQLEditor: React.FC<SQLEditorProps> = ({ onChange, query, language
 };
 
 export const registerLanguageAndSuggestions = async (monaco: Monaco, l: LanguageDefinition, lid: string) => {
+  console.log("registering");
```

Comment:
Use `utils/debugger/sqlEditorLog` if you want a logger that is disabled in production and you don't want to remove console logs before merge :)